### PR TITLE
gcc7 compilation warning(s) fix(es) in HLTrigger/Timer

### PR DESCRIPTION
**Pull request by mrodozov**
Fix for compilation warning(s) when compiling with gcc7 and more flags listed here:
https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc700/CMSSW_9_3_X_2017-07-24-2300/HLTrigger/Timer

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

File: HLTrigger/Timer/test/chrono/src/x86_tsc.cc [hunk 1]
```diff
@@ -163,16 +163,19 @@ extern "C" {
 
   static uint64_t (*serialising_rdtsc_resolver(void))(void)
   {
+
```
Comment (fwyzard) [inline HLTrigger/Timer/test/chrono/src/x86_tsc.cc]: please do not add extra empty lines

File: HLTrigger/Timer/test/chrono/src/x86_tsc.cc [hunk 2]
```diff
@@ -163,16 +163,19 @@ extern "C" {
...
       // if available, use the RDTSCP instruction
       return serialising_rdtscp;
-
+    
+    unsigned eax=0, ebx=0, ecx=0, edx=0;
+    
     if (has_tsc()) {
       // if the TSC is available, chck the processor vendor
-      unsigned int eax, ebx, ecx, edx;
+      //unsigned int eax, ebx, ecx, edx;
```
Comment (fwyzard) [inline HLTrigger/Timer/test/chrono/src/x86_tsc.cc]: why move it out of the `if` block ?

**Comment by fwyzard** [comment]
-1

**Comment by fwyzard** [comment]
please change only line 175 to
```
unsigned int eax = 0, ebx = 0, ecx = 0, edx = 0;
```

**Comment by fwyzard** [comment]
can you
  - not add the empty line 166
  - not add spaces at the end of line 172/173
  - squash all commits into a single one
?

**Comment by davidlange6** [comment]
please test

**Comment by fwyzard** [comment]
-1

**Comment by fwyzard** [comment]
see comments above

**Comment by mrodozov** [comment]
I'm closing this pr find the new one here https://github.com/cms-sw/cmssw/pull/19919

**Pull request closed without merging**